SROS2_Windows.md

```diff
@@ -107,13 +116,13 @@ To do this, we will use the sample policy file provided in `examples/sample_poli
 
 First, we will copy this sample policy file into our keystore:
 
-```bash
-curl -sk https://raw.githubusercontent.com/ros2/sros2/ardent/examples/sample_policy.yaml -o .\demo_keys\policies.yaml
+```bat
+curl -k https://raw.githubusercontent.com/ros2/sros2/ardent/examples/sample_policy.yaml -o .\demo_keys\policies.yaml
```

Comment:
Is this because we want additional printing if the download fail?
Or did something else motivate this change?

Reply:
In my case `-s` was hiding an error saying curl was not built with https support, and I almost didn't notice due to #41. By chance I saw the file wasn't there when deleting the demo_keys directory for another RMW implementation.